run.py

```diff
@@ -47,6 +47,63 @@ def false_if_exc_timeout_or_sub_exists(exception):
 ###############################################################################
 # LOCAL FUNCTION DEFINITIONS
 
+def _copy_files_from_session(fw, from_session, to_session):
+    """Copies file attachments from one session to another
+
+    1. Download each file from "from_session"
+    2. Upload each file to "from_session"
+
+
+    Args:
+        fw (flywheel.Client): A flywheel client
+        from_session (flywheel.Session): An existing flywheel session that has attached files to be
+        copied over to "to_session"
+        to_session (flywheel.Session): An existing flywheel session that will have the files from
+        "from_session" copied over to.
+
+    """
+    log.info(f"Transferring session {from_session.label} attachment files...")
+
+    session_files = [f for f in from_session.files]
+    if len(session_files) == 0:
+        log.info('No files to copy over')
+        return
+
+    for session_file in session_files:
+        
+        if session_file.type == 'dicom':
+            log.warning(f"File {session_file.name} is type DICOM.\n"
+                        f"DICOMS Uploaded as attachments to a session will NOT be migrated,\n"
+                        f"As we cannot properly ensure de-identification at this level.\n"
```

Comment:
```suggestion
                        f"As we do not support DICOM mapping at this level.\n"
```
This gear does not do de-id.